src/odemis/gui/comp/stream_panel.py

```diff
@@ -419,7 +419,12 @@ def _on_colormap_value(self, colour):
         else:  # Can't find the colour => it's custom tint
             if isinstance(colour, tuple):
                 self.colormap_choices[TINT_CUSTOM_TEXT] = colour
-                self.combo_colormap.SetClientData(self._colormap_customtint_idx, colour)
+                # sometimes switching to a custom color will throw a C/C++ wrapper runtime error
+                # this can be ignored and won't affect the colormap selected
+                try:
+                    self.combo_colormap.SetClientData(self._colormap_customtint_idx, colour)
+                except RuntimeError:
```

Comment:
I think that you can keep it generic by just decorating the function with an (extra) `@ignore_dead` (between the call_in_wx_main and the function name).